packages/web-components/examples/codesandbox/components/content-group-horizontal/index.html

```diff
@@ -0,0 +1,53 @@
+<!--
+@license
+
+Copyright IBM Corp. 2020
+
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+  <head>
+    <title>@carbon/ibmdotcom-web-components example</title>
+    <meta charset="UTF-8" />
+    <script type="module" src="src/index.js"></script>
+    <style type="text/css">
+      /* Avoids FOUC - Will be overriden by `index.scss` */
+      body {
+        visibility: hidden;
+      }
+    </style>
+  </head>
+  <body>
+    <div class="bx--grid">
+      <div class="bx--row">
+        <div class="bx--col-sm-4 bx-col-lg-12 bx--offset-lg-4">
+          <dds-content-group-horizontal>
+            <dds-content-block-heading>Curabitur malesuada varius mi eu posuere</dds-content-block-heading>
+            <dds-content-item-horizontal .copy='Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin.'>
+              <slot slot="eyebrow">Lorem ipsum</slot>
```

Comment:
`<slot>` is for defining a slot in shadow DOM, instead of using it. Please refer to: #4406

```suggestion
              <span slot="eyebrow">Lorem ipsum</span>
```